applications/StructuralMechanicsApplication/custom_elements/shell_thick_element_3D3N.cpp

```diff
@@ -472,75 +472,6 @@ void ShellThickElement3D3N::CalculateOnIntegrationPoints(const Variable<double>&
     OPT_INTERPOLATE_RESULTS_TO_STANDARD_GAUSS_POINTS(rValues);
 }
 
-void ShellThickElement3D3N::CalculateOnIntegrationPoints(const Variable<Vector>& rVariable,
-    std::vector<Vector>& rValues,
-    const ProcessInfo& rCurrentProcessInfo)
-{
-    const SizeType num_gps = GetNumberOfGPs();
-    if (rVariable == LOCAL_AXIS_1)
-    {
-        // LOCAL_AXIS_1 output DOES NOT include the effect of section
-        // orientation, which rotates the entrire element section in-plane
-        // and is used in element stiffness calculation.
-
-        // resize output
-        if (rValues.size() != num_gps)
-            rValues.resize(num_gps);
-
-        for (SizeType i = 0; i < num_gps; ++i) rValues[i] = ZeroVector(3);
-        // Initialize common calculation variables
-        ShellT3_LocalCoordinateSystem localCoordinateSystem(mpCoordinateTransformation->CreateReferenceCoordinateSystem());
-
-        for (SizeType GP = 0; GP < 1; GP++)
-        {
-            rValues[GP] = localCoordinateSystem.Vx();
-        }
-    }
-    else if (rVariable == LOCAL_MATERIAL_ORIENTATION_VECTOR_1)
-    {
-        // LOCAL_MATERIAL_ORIENTATION_VECTOR_1 output DOES include the effect of
-        // section orientation, which rotates the entrire element section
-        // in-plane and is used in the element stiffness calculation.
-
-        // resize output
-        if (rValues.size() != num_gps)
-            rValues.resize(num_gps);
-
-        for (SizeType i = 0; i < num_gps; ++i) rValues[i] = ZeroVector(3);
-
-        // Initialize common calculation variables
-        // Compute the local coordinate system.
-        ShellT3_LocalCoordinateSystem localCoordinateSystem(mpCoordinateTransformation->CreateReferenceCoordinateSystem());
-
-        // Get local axis 1 in flattened LCS space
-        Vector3 localAxis1 = localCoordinateSystem.P2() - localCoordinateSystem.P1();
-
-        // Perform rotation of local axis 1 to fiber1 in flattened LCS space
-        Matrix localToFiberRotation = Matrix(3, 3, 0.0);
-        double fiberSectionRotation = mSections[0]->GetOrientationAngle();
-        double c = std::cos(fiberSectionRotation);
```

Comment:
const, and not very descriptive variables names

Reply:
I am removing this ...